core/ui/setting/impl-dialog/src/main/kotlin/net/thunderbird/core/ui/setting/dialog/ui/components/dialog/value/TextDialogView.kt

```diff
@@ -43,20 +44,34 @@ internal fun TextDialogView(
             ),
         )
     }
+    var errorMessage by remember { mutableStateOf<String?>(null) }
```

Comment:
I guess using a `derivedStateOf` here instead will fit better with what you want.

```suggestion
    var errorMessage by remember {
        derivedStateOf {
            val transformedText = setting.transform(textFieldValue.text)
            setting.validate(transformedText)
        }
    }
```